tensorboard/webapp/metrics/views/card_renderer/scalar_card_fob_test.ts

```diff
@@ -89,56 +97,14 @@ describe('ScalarFobController', () => {
     });
   });
 
-  describe('getHighestStep', () => {
-    it('gets the highest step when minMax is in order', () => {
-      const fixture = createComponent({minMax: [0, 2]});
-
-      const highestStep = fixture.componentInstance.getHighestStep();
-
-      expect(highestStep).toBe(2);
-    });
-
-    it('gets the highest step when minMax is not in order', () => {
-      const fixture = createComponent({minMax: [2, 0]});
-
-      const highestStep = fixture.componentInstance.getHighestStep();
+  it('gets the highest/lowest step', () => {
+    const fixture = createComponent({minMaxStep: {minStep: 0, maxStep: 2}});
 
-      expect(highestStep).toBe(2);
-    });
-
-    it('returns the next lowest step when max is a floating point.', () => {
-      const fixture = createComponent({minMax: [4.5, 1]});
-
-      const lowestStep = fixture.componentInstance.getHighestStep();
-
-      expect(lowestStep).toBe(4);
-    });
-  });
```

Comment:
Why is this test being deleted?

Reply:
Fair question, they are removed because they tests the old logic, which used minMax to get he highest/lostest step, which are tested in the new test.

The part of min max step being floating point was an incorrect concept in the new logic (which ideally should not happen) I can 
1. add Math.round function in getHighestStep/getLowestStep if that's necessary (floating point was not tested for the original minMax
2. add minMax being float to function getStepLowerThanAxisPosition but I think that does not matter for testing that function